src/ResponsiveProvider.js

```diff
@@ -139,6 +139,14 @@ const ResponsiveProvider = ({
     );
 };
 
+const breakpointsPropTypes = {
```

Comment:
Are all of these keys always expected, and if so, are they always these exact keys?

If this is true, do you think we should update the `initialMediaType` to use `oneOf` and then these keys instead of `PropTypes.string`, and also update defaultOrientation to use` oneOf(["portrait", "landscape"])` ?  Just since you're already changing the PropTypes anyway.

Reply:
Great points. I'll change that.

Reply:
Nice, thanks!